### ISSUE TITLE
Parse Error: expected: ',' OR new line got: ')'. at ');\n'

### ISSUE THREAD

**Issue by mizanxali** [tdwesten/spotify-charts-com#1]
I've been trying to use this package to scrape data from Spotify charts but I keep getting this error. I am attaching the screenshot below.

**Comment by mizanxali** [comment]
![Screenshot (2)](https://user-images.githubusercontent.com/59915742/126047113-655588d2-da8f-46c6-ade7-48871575861a.png)

**Comment by tdwesten** [comment]
Hi @mizanxali, thanks for sharing this. I will look in to it tomorrow.

**Comment by mizanxali** [comment]
I believe I have figured out what the issue is. Spotify Charts have added a layer of Cloudflare DDOS protection so the .csv file is no longer accessible through an HTTP request.

**Comment by tdwesten** [comment]
Yeah, i see. The site it self is also protected so for now i don't see any options to scrape the charts :/. I will remove this package from the NPM repo. 

Thanks!

**Comment by mizanxali** [comment]
You could use something like [ScraperAPI](https://www.scraperapi.com/) but I believe that would cost you and wouldn't be worth it. Just thought of letting you know.
